docs/CODE_OF_CONDUCT.md

```diff
@@ -76,6 +76,29 @@ address, posting via an official social media account, or acting as an appointed
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
+### AI Generated Contributions Policy
+
+Due to the burden put on maintainers, users submitting multiple low quality pull
+requests, or AI generated comments, reviews, issues, or pull requests, where the
+user does not show a good understanding of what they are posting, might be banned
+from the organisation. Some examples of poor etiquette are:
+
+- Opening a PR without a corresponding issue or for issues which are not yet triaged
```

Comment:
I'd split this in two so that the bullets are all roughly the same "size"
```
- Opening a PR without a corresponding issue 
- Opening a PRs for un-triaged issues
```